npm/deploy/npm/azure-npm.yaml

```diff
@@ -99,18 +99,12 @@ spec:
           volumeMounts:
           - name: xtables-lock
             mountPath: /run/xtables.lock
-          - name: log
-            mountPath: /var/log
           - name: protocols
             mountPath: /etc/protocols
           - name: azure-npm-config
             mountPath: /etc/azure-npm
       hostNetwork: true
       volumes:
-      - name: log
```

Comment:
wondering why we are removing /var/log?

Reply:
@matmerr said we are using this in production anymore. I made an assumption it wasn't being used at all. I will add it back in :)

Reply:
Mat said we are or aren't?

Reply:
We are NOT logging to a file anymore so we can safely remove this.